src/api/posts/src/routes/posts.js

```diff
@@ -60,14 +60,26 @@ posts.get('/', validatePostsQuery(), async (req, res, next) => {
     first: `/posts?per_page=${perPage}&page=${1}`,
     last: `/posts?per_page=${perPage}&page=${Math.floor(postsCount / perPage)}`,
   });
-  res.json(
-    ids
+  let data;
+  if (expand === 1) {
+    data = await Promise.all(
```

Comment:
This can fail, needs `try/catch`

Reply:
Thank for the suggestion, a `try/catch` block is now added to wrap `Promise.all()`.

Reply:
When you say "is now added," there is nothing here.  Please push your changes.

Reply:
Sorry, I forgot to push my code, it's now pushed.